### ISSUE TITLE
Unable to install on mac

### ISSUE THREAD

**Issue by SweetPeach99** [sorayuki/obs-multi-rtmp#197]
Hi!

I tried following the install instructions for the latest macOS version that I found on kilinbox.net but creating a /plugins directory and copying the plug-in there did nothing for me. I also tried copying the plug-in into a folder named plugin_config which already contained some other plugins I use for OBS. Nothing has worked and I have triple checked that I'm on the correct version of OBS and restarted the software after each installation attempt. Is there an installation wizzard/package available for the latest version of OBS? Any help?
<img width="652" alt="Screenshot 2022-09-16 at 12 57 11" src="https://user-images.githubusercontent.com/113685258/190624117-07baed70-7363-4b65-9bf6-6fa4dda49c00.png">

**Comment by keithskchong** [comment]
Not sure am I using the right way to do so... I right click the OBS -> show content, look for the folder Contents -> PlugIns, I put the obs-multi-rtmp.plugin in, start OBS and the dock shown. hope this help.

<img width="342" alt="Screenshot 2022-09-25 at 6 12 51 PM" src="https://user-images.githubusercontent.com/66323494/192138473-a492562b-3bd8-447b-ac10-c74549bb680d.png">
<img width="700" alt="Screenshot 2022-09-25 at 6 11 04 PM" src="https://user-images.githubusercontent.com/66323494/192138474-17867086-b9b3-439c-ab2a-aadb6a7e4f23.png">
<img width="540" alt="image" src="https://user-images.githubusercontent.com/66323494/192138580-b731db75-aa66-4bff-aa4a-e6fdc40c9e94.png">

**Comment by fulldecent** [comment]
Also I see

<img width="372" alt="Screen Shot 2022-10-08 at 22 51 57" src="https://user-images.githubusercontent.com/382183/194735440-6368bb2f-7cb3-4921-a0e5-c7ee83eb7f3d.png">

**Comment by sorayuki** [comment]
Hello, would you like to open an issue at kilinbox's github repository? I'm not falimiar in macOS associated.
________________________________

**Comment by frankbasti** [comment]
Same.  Is there a proper procedure for mac install?

**Comment by astolfo71** [comment]
> Not sure am I using the right way to do so... I right click the OBS -> show content, look for the folder Contents -> PlugIns, I put the obs-multi-rtmp.plugin in, start OBS and the dock shown. hope this help.
> 
> <img alt="Screenshot 2022-09-25 at 6 12 51 PM" width="342" src="https://user-images.githubusercontent.com/66323494/192138473-a492562b-3bd8-447b-ac10-c74549bb680d.png">
> 
> <img alt="Screenshot 2022-09-25 at 6 11 04 PM" width="700" src="https://user-images.githubusercontent.com/66323494/192138474-17867086-b9b3-439c-ab2a-aadb6a7e4f23.png">
> 
> <img alt="image" width="540" src="https://user-images.githubusercontent.com/66323494/192138580-b731db75-aa66-4bff-aa4a-e6fdc40c9e94.png">

You're right man I did the same and now works...I don't know why but I already try this way and doesn't works, after a full uninstall and reinstall obs again, now it works..
